### PR TITLE
always wget from the ceph dumpling branch

### PR DESCRIPTION
**Pull request by dachary**
Signed-off-by: Loic Dachary <ldachary@redhat.com>

### CODE REVIEW

**Comment by dachary** [comment]
@jdurgin @dillaman most of the changes are in rbd related tests. Do you see any of them that would actually require fetching from the ceph master branch instead of the dumpling branch ?

File: tasks/chef.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def task(ctx, config):
 #                '-q',
                 '-O-',
 #                'https://raw.github.com/ceph/ceph-qa-chef/master/solo/solo-from-scratch',
-                'http://ceph.com/git/?p=ceph-qa-chef.git;a=blob_plain;f=solo/solo-from-scratch;hb=HEAD',
+                'http://ceph.com/git/?p=ceph-qa-chef.git;a=blob_plain;hb=dumpling;f=solo/solo-from-scratch;hb=HEAD',
```
Comment (zmc) [inline tasks/chef.py]: ceph-qa-chef has no dumpling branch

**Comment by jdurgin** [comment]
Looks good to me once zack's comment is fixed

**Comment by dachary** [comment]
@zmc @josh fixed & repushed & merged, thanks :-)

**Pull request merged**